.build/check_license.sh

```diff
@@ -9,7 +9,7 @@ run_uber_licence() {
   # Ok, somebody hasn't run `npm install -g uber-licence`, that's cool
   if ! which "$bin" >/dev/null ; then
     npm install "$bin" >/dev/null
-    bin="./node_modules/uber-licence/bin/licence"
```

Comment:
i added a fix in the script. you can add `node_modules/.bin` to your `$PATH` if you want it to work outside the script.

the other option would be to add `-g` when you run `npm install`.